wordpress-mobile/WordPress-iOS
# Properly Placed Pages: new page cell

This PR updates the table view cell for the Page list, adding the featured image and the time stamp. Also the badges are replaced by a normal label.

![simulator screen shot - iphone x - 2019-01-14 at 17 21 21](https://user-images.githubusercontent.com/912252/51131266-28465580-1827-11e9-962b-a13a00b37984.png)

**To Test:**
- Select a blog and open the pages list
- Set a page as Private and/or Review Required to see the badge label
- Set a feature image to see it on the pages list

Update release notes:
